docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -174,3 +174,23 @@ const cleanText = (string) =>
 // Example of a test using the helper
 expect(cleanText(input.value)).to.equal('04-17-2022');
 ```
+
+## Overriding `slots` and `slotProps`
+
+**Slots** are the internal building blocks of the components. These internal elements can be overriden using the `slots` prop, allowing for easy customization of Date and Time Pickers. You can also pass additional props to the available `slots`, using the `slotProps` prop.
+
+The list of available `slots` for each component can be found in the [API References](/x/api/date-pickers/).
+
+You can easily cutomize simple slots as the `openPickerIcon` just by replacing the slot like this:
+
+```jsx
+<DatePicker slots={{ openPickerIcon: EditCalendarRoundedIcon }} />
+```
+
+The subcomponents of the pickers are often built on more nested structures, and you can go as deep into the customization of the nested elements as you want. For instance, you can customize the `IconButton` used as the `openPickerButton`, as well as the `openPickerIcon` inside of it. You can do the same for the `field` and the `textField`.
```

Comment:
IMHO, this section is a bit misleading. You could get the impression that we expose every single component as a slot, even though it's far from the truth. 🤔 
I don't have a much better suggestion for this section, but maybe someone else could pitch in? 🤔

Reply:
Hmmm...I see what you mean. Maybe including something like 'some of which can be overridden'? To make it clear that not everything can be replaced 🤔 I don't really have a better idea
cc. @flaviendelangle @joserodolfofreitas

Reply:
Maybe we can take it from another angle

> Some parts of the UI contains several nested slots. For instance, the adornment of the `TextField` contains 3 slots ([`inputAdornment`, `openPickerButton`, `openPickerIcon`](/x/react-date-pickers/custom-opening-button/) that you can use depending on the React element you are trying to custimize.

Reply:
WDYT about the current change?